### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Michael21107**
grammar and style fixes and improvements

### CODE REVIEW

**Comment by jullevistrunz** [comment]
Thanks for your help, but please pr to the dev branch.

File: README.md [hunk 1]
```diff
@@ -28,24 +28,24 @@ A Police Computer Plugin and Server for LSPDFR.
 ## Citation and Arrest Options
 
 - Files: `EPC/citationOptions.json` & `EPC/arrestOptions.json`
-- `minFine`: Minimum fine in $ for charge
-- `maxFine`: Maximum fine in $ for charge
-- `minMonth`: Minimum jail time in month for charge (arrests only)
-- `maxMonth`: Maximum jail time in month for charge (arrests only), `null` for life in prison
+- `minFine`: Minimum fine in $ for a charge
```
Comment (jullevistrunz) [inline README.md]: It might be clearer to use the charge instead of a charge

File: README.md [hunk 2]
```diff
@@ -61,13 +61,13 @@ A Police Computer Plugin and Server for LSPDFR.
 ## Custom Styles
 
 - File: `EPC/custom.css`
-- For simple customization you may change colors, etc. size in the `:root` selector
-- Feel free to also add more css. `EPC/custom.css` will overwrite `EPC/main/styles.css`
+- For simple customization you may change colors, etc., and size in the `:root` selector
```
Comment (jullevistrunz) [inline README.md:64]: I fucked up but you did too 😅
(Either remove etc. and list everything or only list one)

**Review by jullevistrunz — CHANGES_REQUESTED** [review]
A lot of the other stuff comes down to personal preference, in my opinion.

File: README.md (hunk 1 above)
Comment (Michael21107) [reply README.md]: gotcha

**Review by jullevistrunz — CHANGES_REQUESTED** [review]
I don't think we're getting anywhere with this. This pr doesn't include the latest changes of the dev branch, thus reverting my work. I also think it's not very efficient to keep reviewing a damn README file.

**Comment by Michael21107** [comment]
yea sorry, imma do a new one to the dev branch

**Comment by jullevistrunz** [comment]
No worries. Thanks you for taking the time.

**Pull request closed without merging**